index.html

```diff
@@ -317,18 +313,29 @@
                     loading: false
                 },
                 created: function() {
-                    const searchParams = new URLSearchParams(window.location.search);
+                    let searchParams = new URLSearchParams(window.location.search);
                     if (searchParams.has('stream')) {
                         const queryStream = searchParams.get('stream');
-                        if (["stable", "testing", "next", "testing-devel", "next-devel", "bodhi-updates", "developer"].includes(queryStream)) {
+                        if (this.streamList.includes(queryStream)) {
                             this.stream = queryStream;
+                        } else {
+                            searchParams.set('stream', this.stream);
                         }
+                        history.replaceState(null, null, `${window.location.origin}?${searchParams.toString()}`);
```

Comment:
Added a condition where the URL will be updated with default stream parameter. For example `http://localhost:8000/?stream=helloworld` will be replaced with `http://localhost:8000/?stream=testing-devel`.